azure-quantum/azure/quantum/workspace.py

```diff
@@ -257,6 +257,27 @@ def list_jobs(
 
         return result
 
+    def get_targets(self) -> Dict[str, List[str]]:
+        """Returns a dictionary of provider IDs and lists of targets
+        that are available.
+
+        :return: Targets, keyed by provider IDs
+        :rtype: Dict[str, List[str]]
+        """
+        return {
+            provider.id: [
+                target.id for target in provider.targets
+            ] for provider in self._client.providers.get_status()
+        }
+
+    def get_job_quotas(self) -> List[Dict[str, Any]]:

```

Comment:
job_quotas, seems to refer as the quotas of a single job, maybe just keep it `quotas`?